adr/measuring-custom-handlers.md

```diff
@@ -0,0 +1,233 @@
+# Measuring Custom Handlers
```

Comment:
Thanks for including this! Good read and valuable context.

What does `adr` (the directory name) stand for? :slightly_smiling_face:

Reply:
Architecture Decision Record (ADR)